storybook/utils/zeroReports.test.js

```diff
@@ -0,0 +1,256 @@
+/**
+ * Zero Reports utility tests.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { replaceValuesOrRemoveRowForDateRangeInAnalyticsReport } from './zeroReports';
+import mockedReportResponse from '../../assets/js/modules/analytics-4/utils/__fixtures__/mocked-report-multiple-distinct-date-ranges.json';
+
+describe( 'zeroReports', () => {
+	describe( 'replaceValuesOrRemoveRowForDateRangeInAnalyticsReport', () => {
+		const matchesDateRange = ( data, dateRange ) =>
+			data.some( ( value ) => value.value === dateRange );
+
+		it( 'should zero values for date_range_0 when emptyRowBehavior is "zero"', () => {
+			// Create a deep copy of the mocked report
+			const report = JSON.parse( JSON.stringify( mockedReportResponse ) );
+			const result =
+				replaceValuesOrRemoveRowForDateRangeInAnalyticsReport(
+					report,
+					'date_range_0',
+					'zero'
+				);
+
+			// Expect all rows to be present (10 total)
+			expect( result.rows ).toHaveLength( 10 );
+
+			// Check that date_range_0 rows have zeroed metric values
+			const dateRange0Rows = result.rows.filter( ( row ) =>
+				matchesDateRange( row.dimensionValues, 'date_range_0' )
+			);
+
+			dateRange0Rows.forEach( ( row ) => {
+				row.metricValues.forEach( ( metricValue ) => {
+					expect( metricValue.value ).toBe( '0' );
+				} );
+			} );
+
+			// Check that date_range_1 rows are unchanged
+			const dateRange1Rows = result.rows.filter( ( row ) =>
```

Comment:
1 row isn't plural 😅 

```suggestion
			const dateRangeWithOneRow = result.rows.filter( ( row ) =>
```